src/app/combobox/combobox.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="combobox-container"
+     [class.combobox-disabled]="disabled"
+     [class.combobox-selected]="!!selectedItem">
+  <div class="input-group">
+    <input type="text"
+           autocomplete="false"
+           [placeholder]="placeholder"
```

Comment:
There are some UX and accessibility issues with relying on placeholder text to indicate purpose or instruction for an input component. This could be improved by introducing a label element, using aria-describedby on an element in between the label and input, or using aria-label directly on the input. Here's a good article that was circulated around UXD recently that really goes into how to treat this best.